Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Quote.hs

```diff
@@ -104,7 +105,7 @@ getOffers searchRequest = do
       return . sortBy (compare `on` creationTime) $ quotes <> metroOffers <> publicTransportOffers
     Nothing -> do
       quoteList <- runInReplica $ QRentalQuote.findAllBySRId searchRequest.id
-      let quotes = OnDemandCab . SQuote.makeQuoteAPIEntity <$> sortByEstimatedFare quoteList
+      let quotes = OnRentalCab . SQuote.makeQuoteAPIEntity <$> sortByEstimatedFare quoteList
```

Comment:
Why this change?

Reply:
For having this type of json response for rentals : 
(Note the **onRentalCab** key in the json)
Normally when quotes/estimates come it will have key as onDemandCab.
Thats why we added this.
`{
    "estimates": [],
    "toLocation": null,
    "quotes": [
        {
            "onRentalCab": {
                "tripTerms": [],
                "quoteDetails": {
                    "fareProductType": "RENTAL",
                    "contents": {
                        "perExtraKmRate": 10,
                        `

Reply:
Ohh, I thought `OnDemand` was replaced with `onRental`. Fine